vertex-vector-search/workflows/README.md

```diff
@@ -246,10 +247,14 @@ Management](https://cloud.google.com/storage/docs/lifecycle).
 gs://<bucket_name>/<folder_name>/
 ```
 
+The bucket and folder must be created prior to workflow execution.
+
 ##### Vertex Parameters
 
 `vector_search_index_id`: Vertex AI Vector Search Index which needs to be
-updated.
+updated. To find the id, in the Vertex AI page of the [Google Cloud
```

Comment:
To find the `vector_search_index_id`, go to the [Vertex AI Vector Search](https://console.cloud.google.com/vertex-ai/matching-engine) page in the Google Cloud console.

Reply:
Thanks Tina! Done.

Also made a fix to the scheduler command.